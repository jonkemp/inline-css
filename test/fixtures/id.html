<html><link rel="stylesheet" href="id.css" /><body>
<div id="test">woot</div></body></html>
