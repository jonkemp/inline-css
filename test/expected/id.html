<html><head></head><body>
<div id="test" style="display: none;">woot</div>
</body></html>
